casbin/core_enforcer.py

```diff
@@ -22,8 +22,8 @@ class CoreEnforcer:
     auto_save = False
     auto_build_role_links = False
 
-    def __init__(self, model=None, adapter=None, enable_log=False):
-        self.enable_log(enable_log)
+    def __init__(self, model=None, adapter=None):
+        self.logger = logging.getLogger()
         if isinstance(model, str):
```

Comment:
@favipcj Please refer to https://github.com/casbin/casbin/blob/master/enforcer.go#L63

Reply:
@techoner can you please elaborate on what you are asking for or have issue with? I suspect logging in Go is different than logging in Python and as such should be handled differently. In this instance if a user has not set up the root logger logging will not be enabled. This leaves it up to the application owner to set up logging and is not handled by the supporting packages.

Reply:
The third parameter should not be removed.

Reply:
do you want to keep that parameter in the constructor even if we are not going to use it anymore in the code? I can put it back for backwards compatibility, but it would be better not to do it as it's not going to be used anymore.